src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorHelpers.cs

```diff
@@ -0,0 +1,624 @@
+						}
+						else
+						{
+							currentValue = minBound;
+							break;
+						}
+					}
+
+					currentColorName = ColorHelper.ToDisplayName(ColorConversion.ColorFromRgba(ColorConversion.HsvToRgb(currentHsv)));
+				}
+
+				newValue = (startValue + currentValue + startEndOffset) / 2;
+
+				// Dividing by 2 may have gotten us halfway through a single step, so we'll
+				// remove that half-step if it exists.
+				double leftoverValue = Math.Abs(newValue);
+
+				while (leftoverValue > incrementAmount)
+				{
+					leftoverValue -= incrementAmount;
+				}
+
+				newValue -= leftoverValue;
+
+				while (newValue < minBound)
+				{
+					newValue += wrapIncrement;
+				}
+
+				while (newValue > maxBound)
+				{
+					newValue -= wrapIncrement;
+				}
+			}
+
+			return newHsv;
+		}
+
+		public static double IncrementAlphaChannel(
+			double originalAlpha,
+			IncrementDirection direction,
+			IncrementAmount amount,
+			bool shouldWrap,
+			double minBound,
+			double maxBound)
+		{
+			// In order to avoid working with small values that can incur rounding issues,
+			// we'll multiple alpha by 100 to put it in the range of 0-100 instead of 0-1.
+			originalAlpha *= 100;
+
+			const double smallIncrementAmount = 1;
+			const double largeIncrementAmount = 10;
+
+			if (amount == IncrementAmount.Small)
+			{
+				originalAlpha += (direction == IncrementDirection.Lower ? -1 : 1) * smallIncrementAmount;
+			}
+			else
+			{
+				if (direction == IncrementDirection.Lower)
+				{
+					originalAlpha = Math.Ceiling((originalAlpha - largeIncrementAmount) / largeIncrementAmount) * largeIncrementAmount;
+				}
+				else
+				{
+					originalAlpha = Math.Floor((originalAlpha + largeIncrementAmount) / largeIncrementAmount) * largeIncrementAmount;
+				}
+			}
+
+			// If the value has reached outside the bounds and we should wrap, then we'll place the selection
+			// on the other side of the spectrum.  Otherwise, we'll place it on the boundary that was exceeded.
+			if (originalAlpha < minBound)
+			{
+				originalAlpha = shouldWrap ? maxBound : minBound;
+			}
+
+			if (originalAlpha > maxBound)
+			{
+				originalAlpha = shouldWrap ? minBound : maxBound;
+			}
+
+			// We multiplied alpha by 100 previously, so now we want to put it back in the 0-1 range.
+			return originalAlpha / 100;
+		}
+
+		public static async void CreateCheckeredBackgroundAsync(
+			int width,
+			int height,
+			Color checkerColor,
+			List<byte> bgraCheckeredPixelData,
+			IAsyncAction asyncActionToAssign,
+			CoreDispatcher dispatcherHelper,
+			Action<WriteableBitmap> completedFunction)
+		{
+			if (width == 0 || height == 0)
+			{
+				return;
+			}
+
+			bgraCheckeredPixelData.Capacity = width * height * 4;
+
+			//WorkItemHandler workItemHandler =
+			//(IAsyncAction workItem) =>
+			await Task.Run(() =>
+			{
+				for (int y = 0; y < height; y++)
+				{
+					for (int x = 0; x < width; x++)
+					{
+						//if (workItem.Status == AsyncStatus.Canceled)
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)